api/app/signals/apps/feedback/views.py

```diff
@@ -14,7 +14,8 @@
 class StandardAnswerViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
     """View to list all currently visible Standard Answers."""
     serializer_class = StandardAnswerSerializer
-    queryset = StandardAnswer.objects.filter(is_visible=True).order_by('-id')
+    queryset = StandardAnswer.objects.select_related('topic')\
+        .filter(is_visible=True).order_by('topic__order', 'order')
```

Comment:
Change `.order_by('topic__order', 'order')` to `.order_by('topic__order', 'order', '-id')` to break degeneracies when 'order' or even the combination of 'topic__order' and 'order' is the same for several `StandarAnswers`. This will also retain the ordering for pre-existing `StandardAnswers`.